src/bundle/commitments.rs

```diff
@@ -97,26 +99,46 @@ pub fn hash_bundle_auth_empty() -> Blake2bHash {
 /// Construct the commitment for the issue bundle
 pub(crate) fn hash_issue_bundle_txid_data<A: IssueAuth>(bundle: &IssueBundle<A>) -> Blake2bHash {
     let mut h = hasher(ZCASH_ORCHARD_ZSA_ISSUE_PERSONALIZATION);
+    let mut iah = hasher(ZCASH_ORCHARD_ZSA_ISSUE_ACTION_PERSONALIZATION);
+    let mut ndh = hasher(ZCASH_ORCHARD_ZSA_ISSUE_NOTE_PERSONALIZATION);
```

Comment:
let's have
```rust
let mut ia = hasher(ZCASH_ORCHARD_ZSA_ISSUE_ACTION_PERSONALIZATION);
let mut in = hasher(ZCASH_ORCHARD_ZSA_ISSUE_NOTE_PERSONALIZATION);

```

Reply:
`in` is a keyword, so I used `ia` and `ind` instead.